src/scripts/eastern_kingdoms/silverpine_forest/silverpine_forest.cpp

```diff
@@ -249,16 +253,34 @@ struct npc_deathstalker_faerleiaAI : ScriptedAI
         m_bEventStarted = false;
     }
 
-    void JustRespawned() override
+    void JustRespawned() override {}
```

Comment:
Why override it at all if you're not doing anything on respawn? This line should be removed.